900.version-fixes/m.yaml

```diff
@@ -259,6 +259,7 @@
 - { name: msgpack,                                                                         noscheme: true } # no way to tell whether it's c or cpp counterpart
 - { name: msgpuck,                     ver: "2.0.10",                ruleset: fedora,      incorrect: true }
 - { name: msgpuck,                                                   ruleset: fedora,      untrusted: true }
+- { name: msieve,                      verpat: "[0-9]{4,}",                                snapshot: true } # macports and nix
```

Comment:
That should be `incorrect` instead of `snapshot`. `snapshot` is for e.g. upstream version with date or revision appended, but when it's not based on upstream version it's just incorrect.